src/test/java/org/jabref/logic/journals/JournalAbbreviationRepositoryTest.java

```diff
@@ -134,4 +146,61 @@ void getFromFullName() {
     void getFromAbbreviatedName() {
         assertEquals(new Abbreviation("American Journal of Public Health", "Am. J. Public Health"), repository.get("Am. J. Public Health").get());
     }
+
+    @Test
+    void testAbbreviationsWithEscapedAmpersand() {
+        assertEquals(new Abbreviation("ACS Applied Materials & Interfaces", "ACS Appl. Mater. Interfaces"), repository.get("ACS Applied Materials & Interfaces").get());
+        assertEquals(new Abbreviation("ACS Applied Materials & Interfaces", "ACS Appl. Mater. Interfaces"), repository.get("ACS Applied Materials \\& Interfaces").get());
+        assertEquals(new Abbreviation("Antioxidants & Redox Signaling", "Antioxid. Redox Signaling"), repository.get("Antioxidants & Redox Signaling").get());
+        assertEquals(new Abbreviation("Antioxidants & Redox Signaling", "Antioxid. Redox Signaling"), repository.get("Antioxidants \\& Redox Signaling").get());
+
+        repository.addCustomAbbreviation(new Abbreviation("Long & Name", "L. N.", "LN"));
+        assertEquals(new Abbreviation("Long & Name", "L. N.", "LN"), repository.get("Long & Name").get());
+        assertEquals(new Abbreviation("Long & Name", "L. N.", "LN"), repository.get("Long \\& Name").get());
```

Comment:
Normally, we expect one assertation per test case. Thus, this method should be split into several ones. `@ParamterizedTest` is the the thign to use.